.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: run-tests
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 15.x]
```

Comment:
I didn't see a node version specified in the project, so I'm assuming that we're just using node as a script runner. I just picked the latest versions.